### ISSUE TITLE
Unable to match test files recursively

### ISSUE THREAD

**Issue by scriptdaemon** [tapjs/tapjs#339]
In my project, I use the same directory structure in my test folder as I do my root folder. This looks like the following:

```
bin/bin-file.js
lib/plugins/plugin-file.js
lib/lib-file.js
lib/other-lib-file.js
test/bin/bin-file.test.js
test/lib/plugins/plugin-file.test.js
test/lib/lib-file.test.js
test/lib/other-lib-file.test.js
test/.eslintrc
```

I could very well be (and probably am) misunderstanding glob, but I can't seem to get all the appropriate files to test.

When I try with `tap test/*.test.js`, it doesn't match anything. When I try `tap test/**/*.test.js`, it only matches files one directory deep (i.e. everything in test/bin and test/lib, but not test/bin/plugins). Is this a bug, or am I doing it wrong?

**Comment by isaacs** [comment]
You're probably using a version of Bash that doesn't support globstar, and since the pattern isn't quoted, the shell unpacks the args before tap ever gets them.

Try `tap "test/**/*.test.js"` instead, with quotes.  You can also do `tap test` and it'll walk the dir running all the files in it.

**Comment by isaacs** [comment]
Another fix would be to install Bash version 4.3, and put `shopt -s globstar` in your .bashrc or .profile or .bash_profile files.
